config.json

```diff
@@ -8,5 +8,6 @@
     "enableSejongReply": true,
     "BALId": "819398172270264351",
     "BATId": "888439837256994823",
-    "BAGId": "848725336366645259"
-}
\ No newline at end of file
+    "BAGId": "848725336366645259",
+    "commandDirectories": ["./commands/miscellaneous", "./commands/reminder", "./commands/roles"]
```

Comment:
Need to add the Sejong directory to here once those updated commands are in this branch. Otherwise, they won't get deployed.